AmplifyPlugins/Geo/AWSLocationGeoPlugin/Dependency/AWSDeviceTracker.swift

```diff
@@ -11,16 +11,29 @@ import Foundation
 import CoreLocation
 
 class AWSDeviceTracker: NSObject, CLLocationManagerDelegate, DeviceTrackingBehavior {
-    let locationManager: Geo.LocationManager
+    let locationManager: CLLocationManager
     var wakeAppForSignificantLocationUpdates = true
     var trackUntil: Date = .distantFuture
     var batchingOptions: Geo.LocationManager.BatchingOptions = .none
+    var proxyDelegate: ProxyDelegate?
     
-    init(locationManager: Geo.LocationManager) {
+    init(locationManager: CLLocationManager) {
         self.locationManager = locationManager
     }
     
     func configure(with options: Geo.LocationManager.TrackingSessionOptions) {
+        self.wakeAppForSignificantLocationUpdates = options.wakeAppForSignificantLocationChanges
+        self.trackUntil = options.trackUntil
+        self.batchingOptions = options.batchingOptions
+        self.proxyDelegate = options.proxyDelegate ?? AWSDeviceTrackerProxyDelegate(deviceTrackingBehavior: self)
```

Comment:
what is the reason or need for the default `AWSDeviceTrackerProxyDelegate`? Can it be nil by default?